src/components/moments-ago.vue

```diff
@@ -66,10 +65,12 @@ export default {
     }
   },
 
-  mounted() {
-    setInterval(() => {
-      this.getSeconds(this.date);
+  created() {
+     const interval = setInterval(() => {
```

Comment:
@fieteboerner Can you also add
```js
this.updateDifference();
```
just after line 68? That way the code doesn't rely on the timer to run for the first time

Reply:
yes you're right. now it is updated

Reply:
Awesome, thanks! 👍